Library/ViewModels/BackerDashboardViewModel.swift

```diff
@@ -174,6 +174,18 @@ public final class BackerDashboardViewModel: BackerDashboardViewModelType, Backe
       .map { headerTopConstant, scrollViewYOffset in
         min(headerTopConstant, -scrollViewYOffset)
       }
+
+    self.currentSelectedTabProperty
+      .signal
+      .filter { $0 == .saved }
+      .observeValues { _ in
+        let params = DiscoveryParams.defaults |> DiscoveryParams.lens.starred .~ true
+        AppEnvironment.current.ksrAnalytics.trackDiscoveryModalSelectedFilter(
```

Comment:
The name of this event does not map to the location. This is not the `Discovery` tab. I would create a new method that performs the same event and re-factor the event itself so it doesn't need anything passed into it but params.